storage-bigtable/src/access_token.rs

```diff
@@ -16,17 +16,23 @@ use {
     },
 };
 
-fn load_credentials() -> Result<Credentials, String> {
-    // Use standard GOOGLE_APPLICATION_CREDENTIALS environment variable
-    let credentials_file = std::env::var("GOOGLE_APPLICATION_CREDENTIALS")
-        .map_err(|_| "GOOGLE_APPLICATION_CREDENTIALS environment variable not found".to_string())?;
-
-    Credentials::from_file(&credentials_file).map_err(|err| {
-        format!(
-            "Failed to read GCP credentials from {}: {}",
-            credentials_file, err
-        )
-    })
+fn load_credentials(fp: Option<String>) -> Result<Credentials, String> {
```

Comment:
```suggestion
fn load_credentials(filepath: Option<String>) -> Result<Credentials, String> {
```
Generally, we prefer to spell things out for clarity.